recipes/mtcnn/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "mtcnn" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 56624f08e088e5cf062de8005df09696cdbc0f4c2fc3578483b19e0fb3984252
+
+build:
+  skip: True  # [py<34]
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - keras >=2.0.0
+    - opencv >=4.1
+    - pip
+    - python
+  run:
+    - keras >=2.0.0
+    - opencv >=4.1
+    - python
+
+test:
+  imports:
+    - mtcnn
+    - mtcnn.exceptions
+    - mtcnn.network
+  requires:
+    - nose
+
+about:
+  home: http://github.com/ipazc/mtcnn
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: "Multi-task Cascaded Convolutional Neural Networks for Face Detection, based on TensorFlow"
+  dev_url: http://github.com/ipazc/mtcnn
+
+extra:
+  recipe-maintainers:
+    - Tata17
+    - oblute
+    - benhuff
+    - rluria14
```

Comment:
Could you please confirm that you would like to comaintain this recipe?
@oblute @benhuff @rluria14

Reply:
I will be a maintainer

Reply:
I will also be a maintainer

Reply:
i will also also be a maintainer